dispatcher/helpers/send.py

```diff
@@ -0,0 +1,246 @@
+import pika
+import json
+import random
+import os
+
+"""
+creates between 1 and 8 fake emails and puts in the queue
+OR
+tests all templates it finds in the folder
+"""
```

Comment:
```suggestion

Works by creating a queue that the dispatcher/main.py can read from, without setting up a local RabbitMq. 
"""
```

Reply:
I see that you may think that as I define the queues, I am somehow creating them; but this program simply sends to the queues so we still need a local rabbit. I define the queues because it's how rabbit works

Reply:
indeed, I thought the rabbit was created in-process in send.py, but no, we expect one to exist already. My suggestion is wrongly worded, but it would still be worth to explain that.